### PR TITLE
dev: removed automatic addition multisafeFactoryId 

### PR DESCRIPTION
**Pull request by OlegYak**
Removed automatic addition multisafeFactoryId and added validation to an existing multisafe account.

### CODE REVIEW

File: src/utils/isAccountExist.js [hunk 1]
```diff
@@ -1,14 +1,19 @@
 import { Account } from 'near-api-js';
 
+const code_hash_multiSave = 'EPGksnjsxBjaZkXp63ZqdXK9bFpUzrn4UfW8FrehhRQT'
```
Comment (ilblackdragon) [inline src/utils/isAccountExist.js:3]: let's make this into a list with currently a single item.

Also given this is JS: `multiSafeCodeHashs = [ ' ...' ];`

File: src/utils/isAccountExist.js [hunk 2]
```diff
@@ -1,14 +1,19 @@
...
   if (!accountId) return false;
 
   const connection = near ? near.connection : state.general.entities.near.connection;
 
   try {
-    await new Account(connection, accountId).state();
-    return true;
+    const { code_hash } = await new Account(connection, accountId).state();
+    
+    if(code_hash === code_hash_multiSave) return true;
```
Comment (ilblackdragon) [inline src/utils/isAccountExist.js:13]: check against the list here

**Review by ilblackdragon — APPROVED** [review]
Great, just a quick note on a list

**Review by ilblackdragon — APPROVED** [review]

**Review by agileurbanite — APPROVED** [review]
LGTM

**Pull request merged**